huaweicloud/services/elb/resource_huaweicloud_elb_logtanks.go

```diff
@@ -0,0 +1,144 @@
+package elb
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/elb/v3/logtanks"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func ResourceLogTanksV3() *schema.Resource {
```

Comment:
please remove version part.

Reply:
All other files use this format, maybe it's better to keep consistent here.

Reply:
Those are the old function. If the client version changes in the future, the function name will be inconsistent.

Reply:
I think it can be treated as a later optimization and handled uniformly

Reply:
done